reporter/src/main/kotlin/reporters/AsciiDocTemplateReporter.kt

```diff
@@ -72,6 +72,7 @@ class AsciiDocTemplateReporter : Reporter {
 
         private const val OPTION_BACKEND = "backend"
         private const val OPTION_PDF_THEME_PATH = "pdf-theme.path"
+        private const val OPTION_PDF_FONTS_DIR = "pdf-fontsdir"
```

Comment:
Let's name this `pdf.fonts.dir`.

Reply:
FYI, I've made an alignment PR for `pdf-theme.path` at https://github.com/oss-review-toolkit/ort/pull/3447.